src/utils/contacts.js

```diff
@@ -0,0 +1,34 @@
+import { Map } from 'immutable';
+
+const phoneContact = {
+  name: 'phone',
+  link: '+375293837184',
+  to: 'tel:',
+};
+
+const whatsappContact = {
+  name: 'whatsapp',
+  link: '375293837184',
+  to: 'https://api.whatsapp.com/send?phone=',
+};
+
+const skypeContact = {
+  name: 'skype',
+  link: 'peterbabukh',
+  to: 'skype:',
+};
+
+const emailContact = {
+  name: 'email',
+  link: 'babukhpeter3@gmail.com',
+  to: 'mailto:',
+};
+
+const contactsMap = Map({
+  phone: phoneContact,
+  whatsapp: whatsappContact,
+  skype: skypeContact,
+  email: emailContact,
+});
+
+export default contactsMap;
```

Comment:
also you are likely to have more constants in this file. so better do:
```
export const contactsMap = Map({
  phone: phoneContact,
  whatsapp: whatsappContact,
  skype: skypeContact,
  email: emailContact,
});

export default {
 contactsMap,
}
```

Reply:
fixed